backend/infrahub/workflows/catalogue.py

```diff
@@ -34,7 +34,7 @@
     name="anonymous_telemetry_send",
```

Comment:
I think we sould have a naming convention regarding deployment name to avoid confusion. Should it be the same as flow name or should it be differentiated somehow?

Reply:
Agreed we need to agree on a standard and clean up all the workflows